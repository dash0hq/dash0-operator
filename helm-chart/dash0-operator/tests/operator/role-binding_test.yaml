suite: test role
templates:
  - operator/role-binding.yaml
tests:
  - it: leader election role binding should match snapshot
    asserts:
      - matchSnapshot: {}