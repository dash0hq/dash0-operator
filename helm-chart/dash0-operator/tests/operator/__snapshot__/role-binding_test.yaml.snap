leader election role binding should match snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: RoleBinding
    metadata:
      labels:
        app.kubernetes.io/component: rbac
        app.kubernetes.io/instance: leader-election-rolebinding
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: rolebinding
        app.kubernetes.io/part-of: dash0-operator
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: dash0-operator-1.0.0
      name: dash0-operator-leader-election-rolebinding
      namespace: NAMESPACE
    roleRef:
      apiGroup: rbac.authorization.k8s.io
      kind: Role
      name: dash0-operator-leader-election-role
    subjects:
      - kind: ServiceAccount
        name: dash0-operator-controller-manager
        namespace: NAMESPACE
